lombok.config

```diff
@@ -1,2 +1 @@
-# This file is generated by the 'io.freefair.lombok' Gradle plugin
-config.stopBubbling = true
+lombok.anyConstructor.addConstructorProperties=true
```

Comment:
This is a change that seems out of scope? Do we need it, what is it's benefit?

Reply:
its consistent with the other projects
https://stackoverflow.com/questions/26703645/dont-understand-constructorproperties

we'll need it when we start to use DTOs properly, but we'll forget so a good time to put it in is when we move to the same lombok as the rest of the project.